### DISCUSSION TITLE
Does alipay use oracle mode or mysql mode?

### DISCUSSION THREAD

**Discussion by sprappcom** [oceanbase/oceanbase#1815] [Q&A]
Does alipay use oracle mode or mysql mode?
for financial services, which is better to use and why?

**Comment by hnwyllmm** [comment]
It's not the case. There's only one source code.

**Reply by sprappcom** [reply]
@hnwyllmm no i mean there are two modes for oceanbase, oracle mode and mysql mode.
which is used in alipay?

**Comment by longdafeng** [comment]
It is depending on your application to decide which mode to be used. 
If it is need to be compatible with Oracle, it should use oracle mode, if not, please use mysql mode. 
In alipay, most of applications are using mysql mode. 

But in other finacial companies, their applaciton want to  switch from Oracle to OceanBase, so they perfer to use Oracle mode. 

BTW: OceanBase-CE just provides mysql mode, if you want to use Oracle mode, please contact to the sales for enterprise version.
